ci/run_ctests_gpu.sh

```diff
@@ -13,6 +13,6 @@ fi
 
 # Unless `LEGATE_CONFIG` is set, default to all available GPUs and set fbmem/sysmem.
 # LEGATE_TEST=1 to test broadcasting code paths (locally).
-LEGATE_CONFIG=${LEGATE_CONFIG:- --gpus="$(nvidia-smi -L | wc -l)" --fbmem=2000 --sysmem=6000 --omps=0} \
+LEGATE_CONFIG=${LEGATE_CONFIG:- --gpus="$(nvidia-smi -L | wc -l)" --sysmem=6000 --omps=0} \
```

Comment:
Sorry, hand't noticed this before. Was there much of a reason to remove this?  (Not that it should really matter).